dojo/importers/reimporter/utils.py

```diff
@@ -197,7 +197,7 @@ def get_or_create_product(product_name=None, product_type_name=None, auto_create
 
 
 def get_or_create_engagement(engagement_id=None, engagement_name=None, product_name=None, product_type_name=None, auto_create_context=None,
-                             deduplication_on_engagement=False, source_code_management_uri=None):
```

Comment:
For retro compatibility add the default value here

Reply:
@damiencarol  could you please explain your comment further? I already had target_end=None, it seems like git shows the diff strange, because I changed the line, so it looks like I removed it. This is the reason why I have the check for instance being probably none. Would it be better to have the default value of +365days in the definition?

Reply:
As you are adding a new parameter I was expecting that you put a default value. But it's ok